infra/front-door/front-door.bicep

```diff
@@ -0,0 +1,44 @@
+@description('Required. Name of Front Door Profile.')
+param name string
+
+@description('Required. The pricing tier of the FrontDoor profile.')
+@allowed([
+  'Premium_AzureFrontDoor'
+])
```

Comment:
Maybe should have all the available SKUs / we should not have allowed attribute
The reason is, that if we control the SKU from environment-specific variables, just changing the variable will not have the template working.

Reply:
As discussed, we agreed that the SKU for the front door is not likely to change for any environment.